R/description.R

```diff
@@ -102,3 +106,22 @@ check_package_name <- function(name) {
 valid_name <- function(x) {
   grepl("^[[:alpha:]][[:alnum:].]+$", x) && !grepl("\\.$", x)
 }
+
+tidy_desc <- function(desc) {
+  stopifnot(inherits(desc, "description"), inherits(desc, "R6"))
```

Comment:
Is this how I check that I have an R6 `description` object?

Reply:
Given that this is an internal function, I think you should assume `desc` is already the correct type.